### DISCUSSION TITLE
Can I add dropdown

### DISCUSSION THREAD

**Discussion by phoique** [iddan/react-spreadsheet#207] [Q&A] [answered]
Hello, I really like the package but I need to add dropdown. Is it possible to add custom input fields?

**Comment by iddan** [comment, marked as answer]
Hey, there is an example to how to integrate a dropdown in the stories code:
- https://github.com/iddan/react-spreadsheet/blob/master/src/stories/SelectDataComponents.tsx
- https://github.com/iddan/react-spreadsheet/blob/master/src/stories/Spreadsheet.stories.tsx#L131
